SConstruct

```diff
@@ -429,12 +431,12 @@ def scons():  # pylint: disable=too-many-locals,too-many-branches
 
     if prereqs.client_requested():
         api_version = env.Command("%s/API_VERSION" % build_prefix,
-                                  "%s/SConstruct" % build_prefix,
+                                  "SConstruct",
```

Comment:
I'm having trouble working out this change (unless this is the "input" files) and why it matters.  Is this saying that $TARGET should be re-generated if this filename has changed, and it was previously pointing to an incorrect path so the file wasn't regenerated?

Reply:
Previously, we were setting VariantDir(build_prefix, "."...)

Since we don't do that at this level anymore, I figured it would be better to just use the source file without the build prefix.   This is imperfect way to force API_VERSION to rebuild if the version changes.  Since it's defined in SConstruct, we use it as a source file.